static/js/apps/explore_landing/topics.json

```diff
@@ -561,76 +612,72 @@
       "examples": {
         "general": [
           {
-            "title": "Which countries emit the most greenhouse gases",
-            "url": "oq=Which+countries+emit+the+most+greenhouse+gases"
+            "title": "Which counties emit the most greenhouse gasses",
+            "url": "oq=Which+counties+emit+the+most+greenhouse+gasses&p=Earth&pt=Country&t=dc/topic/GreenhouseGasEmissions"
           },
           {
             "title": "Emissions in Harris County, Texas",
-            "url": "oq=Emissions+in+Harris+County,+Texas"
+            "url": "oq=Emissions+in+Harris+County,+Texas&p=geoId/48201&t=dc/topic/Emissions"
           },
           {
             "title": "Pollution in Los Angeles County",
-            "url": "oq=Pollution+in+Los+Angeles+County"
+            "url": "oq=Pollution+in+Los+Angeles+County&p=geoId/06037&t=dc/topic/Pollutants"
           },
           {
-            "title": "Projected temperature extremes in India",
-            "url": "oq=Projected+temperature+extremes+in+India"
+            "title": "Projected temperature extremes in US counties",
+            "url": "oq=Projected+temperature+extremes+in+USA&p=country/USA&pt=County&t=dc/topic/TemperaturePredictions"
           },
           {
             "title": "Energy use in the US",
-            "url": "oq=Energy+use+in+the+US"
+            "url": "oq=Energy+use+in+the+US&p=country/USA&t=dc/topic/Energy"
           },
           {
             "title": "Green jobs in the US",
-            "url": "oq=Green+jobs+in+the+US"
+            "url": "oq=Green+jobs+in+the+US&p=country/USA&t=dc/topic/GreenJobs"
           }
         ],
         "specific": [
           {
-            "title": "Electronic Waste generated in different countries ",
-            "url": "oq=Electronic+Waste+generated+in+different+countries+"
+            "title": "Electronic Waste generated in different countries",
+            "url": "oq=Electronic+Waste+generated+in+different+countries&p=Earth&pt=Country&t=sdg/EN_EWT_GENPCAP&t=sdg/EN_EWT_GENV"
```

Comment:
Topics/SVs are ordered, so we pack them in a single param:

`t=sdg/EN_WET_GENPCAP___sdg/EN_EWT_GENV`

Reply:
Done!